js/ui/list/ui.list.base.js

```diff
@@ -924,6 +947,22 @@ var ListBase = CollectionWidget.inherit({
         });
     },
 
+    _createGroupHeaderIndicator: function(headerElement) {
+        var that = this;
+
+        $("<div>")
+            .addClass(LIST_GROUP_HEADER_INDICATOR_CLASS)
+            .prependTo(headerElement);
+
+        eventsEngine.on(headerElement, "dxpointerdown", function(e) {
```

Comment:
Use subscription by selector. Making multiple subscriptions can lead to performance issue